### DISCUSSION TITLE
Editor functionalities for keeping a ChangeLog

### DISCUSSION THREAD

**Discussion by JJ** [olivierlacan/keep-a-changelog#487] [Q&A]
I know that Emacs has `add-changelog-entry`. However, I haven't been able to find out if there's something similar in other editors, either native or as a plugin? Can somebody help with Atom, VSCode, vim and so on?

**Comment by tpapp** [comment]
I have a trivial set of functions to open a `CHANGELOG.md` and insert a heading and an `[Unreleased]` template. 

I packaged it up and made it available at https://github.com/tpapp/keepachangelog.el.

Comments, suggestions, PRs are welcome.
